dist/common/scripts/scylla_util.py

```diff
@@ -577,18 +623,29 @@ def is_gentoo_variant():
 def redhat_version():
     return distro.version()
 
-def is_ec2():
-    if os.path.exists('/sys/hypervisor/uuid'):
-        with open('/sys/hypervisor/uuid') as f:
-            s = f.read()
-        return True if re.match(r'^ec2.*', s, flags=re.IGNORECASE) else False
-    elif os.path.exists('/sys/class/dmi/id/board_vendor'):
-        with open('/sys/class/dmi/id/board_vendor') as f:
-            s = f.read()
-        return True if re.match(r'^Amazon EC2$', s) else False
```

Comment:
Why do we need to remove is_ec2() to support GCE?
There are multiple scripts relies on the function, if you remove this you will need to fix them too.

Reply:
right, my bad, deleted it mistakenly when resolving conflicts with Lubush changes

Reply:
fixed